### PR TITLE
Use OSS::Chip in OSS::PowerSelect and OSS::ArrayInput

### PR DESCRIPTION
**Pull request by sBenjamin270495**
### What does this PR do?

Replace code in OSS::PowerSelect and OSS::ArrayInput for use OSS::Chip when you display the items

Related to: https://github.com/upfluence/backlog/issues/1706
### What are the observable changes?

We use OSS::Chip Component in selected-item name block of OSS::PowerSelect so OSS::PowerSelect didn't need the onChange(item, 'deletion') function anymore

### Good PR checklist

- [x] Title makes sense
- [x] Is against the correct branch
- [x] Only addresses one issue
- [x] Properly assigned
- [x] Added/updated tests
- [x] Added/updated documentation
- [ ] Migrated touched components to Glimmer Components
- [x] Properly labeled

### CODE REVIEW

File: addon/components/o-s-s/array-input.hbs [hunk 1]
```diff
@@ -1,11 +1,7 @@
...
-    <div class="input-array-tag fx-row fx-xalign-center">
-      {{item}}
-      <i class="fa fa-remove margin-left-xxx-sm text-color-blue-gray" role="button"
-         {{on "click" (fn this.removeItem index)}}></i>
-    </div>
+    <OSS::Chip @label={{item}} @onRemove={{fn this.removeItem index}} />
   {{/each}}
-  <Input @value={{this.currentValue}} @placeholder={{@placeholder}} {{on "keydown" this.keyListener}}
-         autocomplete="off" {{on "blur" this.validateTagOnClickOutside}} />
+  <Input @value={{this.currentValue}} @placeholder={{@placeholder}} {{on "keydown" this.keyListener}} autocomplete="off"
```
Comment (phndiaye) [inline addon/components/o-s-s/array-input.hbs:5]: can you move the `autocomplete="off"` just after the placeholder arg ? we can see all the `on` modifiers together

File: tests/integration/components/o-s-s/array-input-test.js [hunk 2]
```diff
@@ -69,27 +69,28 @@ module('Integration | Component | OSS::ArrayInput', function (hooks) {
...
     test('WHEN the validator is truthy, the value is added as a tag', async function (assert) {
       validator.returns(true);
-      await render(hbs`<OSS::ArrayInput @validator={{this.validator}}/>`);
+      await render(hbs`<OSS::ArrayInput @validator={{this.validator}} />`);
       await fillInputAndValidate('hashtag');
       assert.ok(validator.calledWith('hashtag'));
       assert.ok(validator.returned(true));
-      assert.dom('.input-array-tag').hasText('hashtag');
+      assert.dom('.upf-chip').exists();
+      assert.dom('.array-input-container').hasText('hashtag');
```
Comment (JulienVannier66) [inline tests/integration/components/o-s-s/array-input-test.js:84]: In your tests, you change `.input-array-tag` into `upf-chip` or `.array-input-container`, there is a reason for this?

File: tests/integration/components/o-s-s/power-select-test.ts [hunk 3]
```diff
@@ -52,9 +52,9 @@ module('Integration | Component | o-s-s/power-select', function (hooks) {
       });
 
       await render(hbs`
-        <OSS::PowerSelect @onSearch={{this.onSearch}}>
+        <OSS::PowerSelect @selectedItems={{this.selectedItems}} @onSearch={{this.onSearch}}>
           <:selected-item as |selectedItem|>
-            {{selectedItem.name}}
+            {{selectedItem}}
```
Comment (JulienVannier66) [inline tests/integration/components/o-s-s/power-select-test.ts:57]: Why changing this test? It fails?

File: tests/integration/components/o-s-s/array-input-test.js (hunk 2 above)
Comment (sBenjamin270495) [reply tests/integration/components/o-s-s/array-input-test.js:84]: .input-array-tag isn't use anymore so i delete it and i've one test to update in publisher admin

**Review by phndiaye — APPROVED** [review]

**Pull request merged**